database/buildSchema/06_template_version.sql

```diff
@@ -1,7 +1,5 @@
 -- template_version table
 
-CREATE TYPE public.template_status AS ENUM ('Draft', 'Available', 'Disabled');
-
```

Comment:
There's a problem deleting this here. I know you put it in in your previous PR (#24), but those changes aren't merged here, so now there's no `template_status` type at all, and the build script failed. 😞

Simplest thing would be to just restore this line here so I can continue with the review, then remove it in #24 as per my review comment, right?

Reply:
Done